### PR TITLE
API to fetch user data 

### PR DESCRIPTION
**Pull request by uditpd3000**

### CODE REVIEW

**Comment by navneet-28** [comment]
closes #11

**Review by namangup — CHANGES_REQUESTED** [review]
Remove scripts folder. Format with black. Will review the code after that.

**Comment by uditpd3000** [comment]
closes #11

File: user/apps.py [hunk 1]
```diff
@@ -2,5 +2,4 @@
 
 
 class UserConfig(AppConfig):
-    default_auto_field = "django.db.models.BigAutoField"
```
Comment (namangup) [inline user/apps.py]: Why is this removed?

File: user/models.py [hunk 2]
```diff
@@ -4,6 +4,7 @@
 class User(models.Model):
     name = models.CharField(max_length=120)
     username = models.CharField(max_length=40, unique=True)
+    roll = models.CharField(max_length=20, unique=True)
```
Comment (namangup) [inline user/models.py]: max_length 10 should be enough

File: user/tests.py [hunk 3]
```diff
@@ -1,3 +1,4 @@
-from django.test import TestCase
+from django.db import reset_queries
+from django.test import TestCase, Client
```
Comment (namangup) [inline user/tests.py]: Any reason for these edits?

File: user/serializers.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
+from rest_framework import fields, serializers
+
+from .models import User
+
+
+class userserializer(serializers.ModelSerializer):
```
Comment (namangup) [inline user/serializers.py]: Class name should be title case `UserSerializer`

File: user/tests.py (hunk 3 above)
Comment (uditpd3000) [reply user/tests.py]: No, actually in the initial stage of using Postman, I didn't know how to do tests so some mishappenings.

File: user/serializers.py (hunk 4 above)
Comment (uditpd3000) [reply user/serializers.py]: I have made all the suggested changes. Please revisit them.

File: user/urls.py [hunk 5]
```diff
@@ -0,0 +1,10 @@
+from django.urls import path, include
+from django.contrib import admin
+from django.views.decorators.csrf import csrf_exempt
+from rest_framework import routers, urlpatterns
+
+from . import views
+
+app_name = "User"
```
Comment (namangup) [inline user/urls.py]: Is this required? I think this can be removed

File: user/utils.py [hunk 6]
```diff
@@ -0,0 +1,9 @@
+from .models import User
+
+
+def IsLoggedIn(request):
+    try:
+        user = User.objects.get(username=request.session["username"])
+        return user
+    except:
+        return None
```
Comment (namangup) [inline user/utils.py]: Include a check `if request.session.has_key("username"):` 
`else: return None`

**Comment by namangup** [comment]
Resolve conflicts and add api documentation to README

**Pull request merged**